.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+ 
+When contributing to this repository, please first discuss the change you wish to make via issue,
+or [Hypertrace slack](https://bit.ly/hypertrace-slack) with the owners of this repository before making a change.
+ 
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+ 
+## Submitting bugs
+ 
+### Due diligence
+-------------
+ 
+Before submitting a bug, please do the following:
+ 
+* Perform **basic troubleshooting** steps:
+ 
+   * **Make sure you're on the latest version.** If you're not on the most
+     recent version, your problem may have been solved already! Upgrading is
+     always the best first step.
+   * **Try older versions.** If you're already *on* the latest release, try
+     rolling back a few minor versions (e.g. if on 1.7, try 1.5 or 1.6) and
+     see if the problem goes away. This will help the devs narrow down when
+     the problem first arose in the commit log.
+   * **Try switching up dependency versions.** If the software in question has
+     dependencies (other libraries, etc) try upgrading/downgrading those as
+     well.
+ 
+* **Search the GitHub issues** to make sure it's not a known
+ issue.
+* If you don't find a pre-existing issue, consider **checking with the slack channel** in case the problem is non-bug-related.
+ 
+### What to put in your issue
```

Comment:
We probably don't need this since we have or will have issue templates.

Also the bullet points do not seem right, most folks will consumer only docker images so probably we need info about the docker/orchestrator.

Reply:
makes sense. addressed.